### PR TITLE
Issue #2889 Update default openshift version to 3.11.0

### LINKED ISSUE #2889: Update default Openshift version from 3.10 to 3.11

### PR DESCRIPTION
**Pull request by praveenkumar**
Fixes #2889 

Steps to test the pull request

  1. 
  2. 
  3. 
  4.

### CODE REVIEW

**Comment by praveenkumar** [comment]
retest this please

**Comment by praveenkumar** [comment]
retest this please.

File: cmd/minishift/cmd/start.go [hunk 1]
```diff
@@ -787,25 +774,17 @@ func populateStartFlagsToViperConfig() {
...
-		}
+	// Set default value for base config for 3.10
+	clusterUpParams["base-dir"] = baseDirectory
+	if viper.GetString(configCmd.ImageName.Name) == "" {
+		imagetag := fmt.Sprintf("'%s:%s'", minishiftConstants.ImageNameForClusterUpImageFlag, config.OpenShiftVersion)
+		viper.Set(configCmd.ImageName.Name, imagetag)
+	}
+	// Add docker bridge subnet to no-proxy before passing to oc cluster up
+	// This will be removed once proxy issue fixed on upstream side for oc cluster up.
+	// https://github.com/openshift/origin/issues/20496
```
Comment (LalatenduMohanty) [inline cmd/minishift/cmd/start.go]: @praveenkumar I do not think this will be ever fixed from oc cluster up side.

Comment (LalatenduMohanty) [reply cmd/minishift/cmd/start.go]: So I guess we can remove the reference.

Comment (praveenkumar) [reply cmd/minishift/cmd/start.go]: @LalatenduMohanty sure, you can remove this comment during merge.

**Comment by LalatenduMohanty** [comment]
Merged https://github.com/minishift/minishift/commit/577d5a44c6c0693fdcdbe203aa8cc72e15225808

**Pull request closed without merging**